.rubocop.yml

```diff
@@ -20,9 +20,9 @@ Rails/Validation:
 Rails/FindBy:
   Enabled: false
 Metrics/LineLength:
-  Max: 155
+  Max: 250
```

Comment:
Not cool. Unless you have a 48'' display)

One hundred should be enough. You can disable this cop for, for example, specs (using `Exclude` option).